client.go

```diff
@@ -66,6 +66,16 @@ func (c *Client) Post(resource string, data url.Values) ([]byte, error) {
 	return c.do(req)
 }
 
+func (c *Client) Put(resource string, data url.Values) ([]byte, error) {
+	req, err := http.NewRequest("PUT", c.endpoint+resource, strings.NewReader(data.Encode()))
+	if err != nil {
+		return nil, err
+	}
+	req.Header.Set("Content-Type", "application/x-www-form-urlencoded")
```

Comment:
In that case I'd suggest to merge this PR containing `[]byte` responses and I'll change the responses in my `dev` branch to objects. Would not increase the scope of this PR and create a clean cut for the next one…